vagrant/config/versions.yaml

```diff
@@ -55,6 +55,18 @@ installers:
       - 'ubuntu1604'
       - 'ubuntu1804'
 
+  - foreman: '2.0'
+    katello: '3.15'
+    pulp: '2.21'
+    puppet: 6
+    boxes:
+      - 'centos7'
+      - 'centos7-fips'
+      - 'debian9'
```

Comment:
This should be dropped since it's not supported. Looks like this should also be reflected in nightly